src/app/core/entity-details/abstract-entity-details/abstract-entity-details.component.ts

```diff
@@ -60,18 +60,29 @@ export abstract class AbstractEntityDetailsComponent implements OnChanges {
 
   protected async loadEntity() {
     this.isLoading = true;
-    if (this.id === "new") {
-      if (this.ability.cannot("create", this.entityConstructor)) {
-        this.router.navigate([""]);
-        return;
+    try {
+      if (this.id === 'new') {
+        if (this.ability.cannot('create', this.entityConstructor)) {
+          this.router.navigate(['']);
+          return;
+        }
+        this.entity = new this.entityConstructor();
```

Comment:
Following best practices, let's only wrap the smallest possible part of the code into a try-catch. I think the part for "new" entities is not related to your issue. It will be enough to just try-catch the `entityMapperService.load` call?

Reply:
Thanks for adjusting this :+1: